docs/classad-attributes/classad-attributes-added-by-collector.rst

```diff
@@ -1,60 +1,46 @@
 ClassAd Attributes Added by the *condor_collector*
 ===================================================
 
-
 :index:`ClassAd attribute added by the condor_collector`
-:index:`AuthenticatedIdentity<single: AuthenticatedIdentity; ClassAd attribute added by the condor_collector>`
 
-``AuthenticatedIdentity``:
+:classad-attribute:`AuthenticatedIdentity`
     The authenticated name assigned by the *condor_collector* to the
     daemon that published the ClassAd.
 
-:index:`AuthenticationMethod<single: AuthenticationMethod; ClassAd attribute added by the condor_collector>`
-
-``AuthenticationMethod``:
+:classad-attribute:`AuthenticationMethod`
     The authentication method used by the *condor_collector* to
     determine the ``AuthenticatedIdentity``.
 
-:index:`LastHeardFrom<single: LastHeardFrom; ClassAd attribute added by the condor_collector>`
-
-``LastHeardFrom``:
+:classad-attribute:`LastHeardFrom`
     The time inserted into a daemon's ClassAd representing the time that
     this *condor_collector* last received a message from the daemon.
     Time is represented as the number of second elapsed since the Unix
     epoch (00:00:00 UTC, Jan 1, 1970). This attribute is added if
     ``COLLECTOR_DAEMON_STATS`` is ``True``.
 
-:index:`UpdatesHistory<single: UpdatesHistory; ClassAd attribute added by the condor_collector>`
+:index:`COLLECTOR_DAEMON_HISTORY_SIZE`
 
-``UpdatesHistory``:
+:classad-attribute:`UpdatesHistory`
     A bitmap representing the status of the most recent updates received
     from the daemon. This attribute is only added if
-    ``COLLECTOR_DAEMON_HISTORY_SIZE``
-
-:index:`COLLECTOR_DAEMON_HISTORY_SIZE` is non-zero. See
+    ``COLLECTOR_DAEMON_HISTORY_SIZE`` is non-zero. See
```

Comment:
Shouldn't this be *:macro:`COLLECTOR_DAEMON_HISTORY_SIZE`*, for the direct link?  (Does doing that autogenerate an index entry?)